src/main/java/com/ncedu/fooddelivery/api/v1/dto/order/CreateOrderDTO.java

```diff
@@ -0,0 +1,59 @@
+package com.ncedu.fooddelivery.api.v1.dto.order;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.ncedu.fooddelivery.api.v1.dto.CoordsDTO;
+import lombok.Data;
+
+import javax.validation.constraints.*;
+import java.util.HashMap;
+import java.util.List;
+
```

Comment:
Аналогично с `CountOrderCostResponseDTO `
```suggestion
   @AllArgsConstructor
```